docs/tenancy/1.x/database-sqlite.md

```diff
@@ -22,3 +55,18 @@ This package will allow you to use the `hooks-database` for SQLite Database inst
 > CAUTION: This will only provide **local** databases, which are not remote accessibly.
 
 > CAUTION: This database driver might require additional steps for configuration. It is based on a filesystem, and not a real database like most Database Drivers.
+
+## Configuration
+
+Detailed configuration steps are located in the [`hooks-database`](hooks-database) package and the [`affects-connections`](affects-connections) package.
+
+### Example
+
+In the following example we set the connection to use SQLite, and set the database path to be specific to the tenant based on the Tenant's ID.
```

Comment:
Let's use in the tenantKey instead, it's a contract function so feels clearer / better